### ISSUE TITLE
ToastObject type is not exported

### ISSUE THREAD

**Issue by builder7777** [shakee93/vue-toasted#118]
When using this plugin in a typescript project, when declaring it in main.ts as such:

```
Vue.use(Toasted, {
  action: {
    text: 'x',
    onClick: (e: HTMLElement, t: ToastObject) => t.goAway(0),
  },
});
```

the **ToastObject** type cannot be found since it's declared only internally in `vue-toasted/types/index.d.ts`

**Comment by Nfinished** [comment]
Solved by #116 ?

**Comment by shakee93** [comment]
Solved
